### ISSUE TITLE
Alert box with quite lengthy content cannot be closed

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#7930]
**Describe the issue you encountered:**
Alert box with quite lenghty content cannot be closed. This could be used by an attacker (despite of low security concern).

- Brave Version (revision SHA): 0.13.5

- Steps to reproduce:
    1. Open https://jsfiddle.net/c5o7arqu/
    2. Make the browser window small
    3. Click "OK"
    
- Actual result: You cannot scroll the alert box to the "OK" button

- Expected result: It should be possible to scroll the alert box to the button

- Screenshot if needed:

- Any related issues: #2775 #7107

**Comment by NejcZdovc** [comment]
Looks ok on macOS, 0.13.5 or maybeI didn't understand the issue

![image](https://cloud.githubusercontent.com/assets/9574457/24401326/3ff6ca62-13b4-11e7-8718-bb4858671274.png)

**Comment by luixxiul** [comment]
It is because the monitor resolution on macOS is higher than Windows. On macOS you can reproduce the issue by adding more text on the devtool.

On Windows 7 on 1152x864 monitor I get this:

![clipboard01](https://cloud.githubusercontent.com/assets/3362943/24401572/e3ba7320-13ef-11e7-9de5-309442559a5f.jpg)

What we would like to have is:

![scroll](https://cloud.githubusercontent.com/assets/3362943/24401641/1a004d2e-13f0-11e7-881c-7a7268afb3f0.gif)

Is it clear?

**Comment by NejcZdovc** [comment]
Got it. We need overflow. Should be easy fix

**Comment by srirambv** [comment]
Its really bad on Windows. This is on maximized window
![image](https://cloud.githubusercontent.com/assets/17010094/24402742/cd9eef78-13b1-11e7-91f8-0017ca2d0428.png)

**Comment by NejcZdovc** [comment]
We should also add some margin on the side

**Comment by bsclifton** [comment]
Good catch! I'll pull this into 0.14.2 😄

**Comment by luixxiul** [comment]
ReleaseNote should be fixed as well:

![clipboard01](https://cloud.githubusercontent.com/assets/3362943/24450941/d87c25ee-14b7-11e7-87e7-76ebd1de7abc.jpg)

STR:
  1. Set BRAVE_UPDATE_VERSION=0.8.3 
  2. Start the browser
  3. Update the browser

Currently ReleaseNote is so simply coded that we should upgrade it in a future: https://github.com/brave/browser-laptop/blob/master/app/renderer/components/releaseNotes.js#L24

**Comment by jonathansampson** [comment]
The dialog also needs to have a max-width, IMHO:

![image](https://cloud.githubusercontent.com/assets/815158/25349621/12fe70fc-28e8-11e7-8571-e1911edd7a42.png)

**Comment by cezaraugusto** [comment]
I know this is supposed to be a low hanging fruit for first-time contribs but this is really bad UI so I'm taking
